docs/technical_specs/model_libraries/gas_solid_contactors/unit_models/fixed_bed_0D.rst

```diff
@@ -0,0 +1,168 @@
+Fixed Bed 0D Reactor
+====================
+
+The IDAES Fixed Bed 0D Reactor (FixedBed0D) model represents a unit operation where a gas stream passes through a 
+solid phase bed.
+The FixedBed0D mathematical model is a 0-D time variant model with two phases (gas and solid). The model captures the
+gas-solid interaction between both phases through reaction, mass and heat transfer.
+
+**Assumptions:**
+
+* There is assumed to be no axial or radial variation in solid composition. 
+* Excess gas flowrate and ideal gas behavior are assumed.
+* The reactor is assumed to be isothermal, and the reaction is one-way (no equilibrium).
+
+**Requirements:**
+
+* Property packages contain temperature and pressure variables.
+
+The FixedBed0D model equations are derived from spacially simplying an existing model of similar geometry and adding time-variant behavior:
+
+A. Ostace, A. Lee, C.O. Okoli, A.P. Burgard, D.C. Miller, D. Bhattacharyya, Mathematical modeling of a moving-bed reactor for chemical looping combustion of methane, in: M.R. Eden, M. Ierapetritou, G.P. Towler (Eds.),13th Int. Symp. Process Syst. Eng. (PSE 2018), Computer-Aided Chemical Engineering 2018, pp. 325–330 , San Diego, CA.
+
```

Comment:
This can be deleted. The 0DFixedBed model isn't derived from the MovingBed model.